### ISSUE TITLE
Can't run my program when console is overloaded.

### ISSUE THREAD

**Issue by EinsMalte** [processing/processing#6230]
## Description
When my program unexpectedly spams the console, the IDE gets overwhelmed with too much data and I can't start my program again

## Expected Behavior
It should start my program again.

## Current Behavior
It is buggy.

## Steps to Reproduce
1. Write crappy code
2. Have too many errors in the console
3. Try restarting your program (optional:) after some fixing

## Your Environment
Processing
* Processing version: I don't know 3.sum
* Operating System and OS version: Windows 10 64 Bit
* Other information:

## Possible Causes / Solutions
Restarting, therefore clearing the console helps.

**Comment by benfry** [comment]
Unfortunately there's not much more we can do here. There are already some things in the code that try to prevent the app from locking up in these cases. But at the end of the day, you're writing a program, and you ask the computer to do too much, then you can also make it crash.

**Comment by dzaima** [comment]
@benfry If stdout and stderr messages are mixed together, there is a bug in Processing in its multithreading handling this. I tried to fix this [here](https://github.com/dzaima/processing4/commit/a380747935cdcdd260bb84576f4e87bd65998106#diff-52f3087e05988640cec320bd60e969baaa6be32c323b67de124d3df921098c41L306) with a lock, but that doesn't seem like a good way to do it. Thoughts?

**Comment by dzaima** [comment]
I linked to the specific part of the `app/src/processing/app/ui/EditorConsole.java` file that has the necessary changes there, but the links can be buggy. (I had to squash the files when the processing 4 history was rewritten to remove large files; probably could've avoided doing that, but didn't bother). `for (int i = 0; i < 10000; i++) { println("out"); System.err.println("err"); }` quickly results in some `java.util.EmptyStackException`s being thrown and running being broken.
